.gitignore

```diff
@@ -54,7 +54,13 @@ buck-out/
 *.jsbundle
 
 # CocoaPods
-Apps/APN/ios/Pods/
+Pods
 
-package-lock.json
-lib/module/index.js
+.Bundle
+
+# Yarn
+.yarn/install-state.gz
+yarn.lock
+
```

Comment:
Moving `yarn.lock` out of `.gitignore` as missing lock file is throwing error when example project is run. [Here is the detail](https://linear.app/customerio/issue/MBL-424/[spike]-review-the-current-react-native-cdp-project-and-draft-a#comment-17dea1f4) if you want to learn more about the yarn error.